client/index.html

```diff
@@ -261,14 +267,14 @@ <h1 class="title">Settings</h1>
 							<div class="col-sm-12">
 								<label for="theme-select" class="sr-only">Theme</label>
 								<select id="theme-select" name="theme" class="input">
-									<% themes.forEach(function(themeName) { %>
-										<option value="<%= themeName %>">
-											<%= themeName.charAt(0).toUpperCase() + themeName.slice(1) %>
+									{{#each themes}}
+										<option value="{{this}}">
+											{{firstLetterUppercase this}}
```

Comment:
I think the capitalized version looks way less sloppy. Since we can't have JS in the template (I believe), it kinda makes sense we need a helper.
@YaManicKill, I would call this `capitalize`, if that makes sense. I can see how we would need such a helper in other places as our code grows.